hnn_core/find_coords.py

```diff
@@ -0,0 +1,16 @@
+"""Temp file to get coords"""
+
+from hnn_core import pyramidal
+
+
+def find_coords(func_name, cell_name):
+    cell = func_name(cell_name=cell_name)
+    # print(cell.sections['soma'].end_pts)
+    for section_name, section in cell.sections.items():
+        print(section_name)
+        print(section.end_pts)
```

Comment:
The `end_pts` printed here are different from those defined in `cells_default` for cell `L2Pyr`. 
![Screenshot from 2023-07-03 01 42 33](https://github.com/jonescompneurolab/hnn-core/assets/63421712/c5dcaa63-d86b-4006-9dc9-11fd68d1da82)

Do I simply need to replace these `end_pts` in `cells_default` or should I build the cell and find them using `cell._nrn_sections`

Reply:
You need to build the cell and get the coords from that

Reply:
When any new cell is created using `Cell` class, a private function `__update_end_pts()` is called at the end of `__init__`. This function creates neuron sections by calling `_create_sections()` then updates the `end_pts` of the `section`. The neuron objects are then deleted using `del self._nrn_sections[name]`.

I have a few questions regarding this flow -
- Neuron objects are created and destroyed in this flow which according to the discussion in last meeting should be avoided outside of simulation.
- If this functionality is already there and the `end_pts` are automatically updated, is manually updating the `end_pts` required ?
- Should this function `_update_end_pts()` be removed?

Reply:
Also if `neuron objects` can be cleared using `del`, can't we follow a similar scheme while saving the network? After calling `build` on the copy of a cell, can we just iteratively delete the `_nrn_sections` and `_nrn_synapses` of the built cell??

Reply:
The problem is that Neuron objects are wrappers around C objects ... when `del` is called, it makes a request to the garbage collector to clear the object from memory. It's a request, not an order. If a pointer is not cleared from memory because of how it is coded in C, it will lead to a segmentation fault and/or slow down the following simulation. The latter will be very hard to debug ... we do have a scheme to clear neuron objects in `NetworkBuilder` but it was contributed by someone really in the weeds who understands Neuron. 

But you have a good point about `_update_end_pts`  ... I think @ntolley implemented that. Not sure how well it's been tested if the `del` in that function does not lead to a performance regression. Any thoughts Nick?